matminer/featurizers/base.py

```diff
@@ -345,7 +345,10 @@ def featurize_many(self, entries, ignore_errors=False, return_errors=False,
             else:
                 tqdm_func = tqdm.tqdm
             # list() required, tqdm has issues with memory if generator given
-            entries = tqdm_func(list(entries), desc=self.__class__.__name__)
+            if sys.version_info[0] < 3 and n_jobs == 1 and pbar == True:
```

Comment:
This should be:

```python
if (n_jobs == 1 or sys.version_info[0] < 3) and pbar:
```

Reply:
We could also just remove this version check as pymatgen no longer officially supports Py2. We could move it to __init__ in root folder maybe so when stuff from matminer is imported and the version is <3, a warning is thrown saying the version is not supported?